OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -51,6 +52,18 @@ public EditorDefaultBrush(EditorViewportControllerWidget editorWidget, WorldRend
 				.ToDictionary(r => r.Info.ResourceType, r => r);
 		}
 
+		long CalculateActorSelectionPriority(EditorActorPreview actor)
```

Comment:
I think you should pass in the world pixel as a parameter, like you're doing in #10011 (maybe rename it too, for consistency).

Will also be worth using the shared method you've created in that PR for calculating the priority.